plugins/defaultplugins/l2plugin/vppdump/dump_vppcalls.go

```diff
@@ -102,10 +102,23 @@ func DumpBridgeDomains(log logging.Logger, vppChan vppcalls.VPPChannel, timeLog
 			return nil, err
 		}
 
+		log.Warnf("tag %v, learn %v, forward %v, flood %v, arpT %v, uuflood %v, macage %v,",
+			bdDetails.BdTag, bdDetails.Learn, bdDetails.Forward, bdDetails.Flood, bdDetails.ArpTerm, bdDetails.UuFlood, bdDetails.MacAge)
+		log.Warnf("id %v, ifcount %v, if %v, bvi %v", bdDetails.BdID, bdDetails.NSwIfs, bdDetails.SwIfDetails, bdDetails.BviSwIfIndex)
+
 		// bridge domain details
 		bds[bdDetails.BdID] = &BridgeDomain{
 			Interfaces: []*BridgeDomainInterface{},
 			BridgeDomains_BridgeDomain: l2nb.BridgeDomains_BridgeDomain{
+				Name: 				 func(input[]byte) string {
```

Comment:
It seems to me that this function could be replaced with `string(bytes.Replace(input, []byte{0x00}, []byte{}, -1))`.